server/src/main/java/org/opensearch/rest/RestController.java

```diff
@@ -95,6 +87,8 @@ public class RestController implements HttpServerTransport.Dispatcher {
     private static final DeprecationLogger deprecationLogger = DeprecationLogger.getLogger(RestController.class);
     private static final String OPENSEARCH_PRODUCT_ORIGIN_HTTP_HEADER = "X-opensearch-product-origin";
 
+    private static final String OPENSEARCH_AUTHENTICATION_TOKEN_HEADER = "_opensearch_auth_token";
```

Comment:
would this variable be used in this class?

Reply:
No, it should be moved.

Reply:
Removed